roles/sarek/templates/DELIVERY.README.SAREK.WES.md.j2

```diff
@@ -0,0 +1,98 @@
+# DELIVERY OF RESULTS FROM EXOME ANALYSIS WITH SAREK
```

Comment:
This file needs to be generated in the tasks.yml file in the sarek role, like here
https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/234a129597be23bb8f37064be81f9a87b616d53a/roles/sarek/tasks/main.yml#L56